rest_framework_simplejwt/serializers.py

```diff
@@ -141,4 +146,15 @@ class TokenVerifySerializer(serializers.Serializer):
     def validate(self, attrs):
         UntypedToken(attrs['token'])
 
+        if 'rest_framework_simplejwt.token_blacklist' in settings.INSTALLED_APPS:
```

Comment:
Try this instead of checking INSTALLED_APPS: https://github.com/SimpleJWT/django-rest-framework-simplejwt/blob/master/rest_framework_simplejwt/serializers.py#L105-L112 It's a little better checking if blacklist is even turned on rather than checking if the app is installed.